readme.html

```diff
@@ -90,47 +90,66 @@ <h2>Port Details</h2>
           <div class="body" role="main">
 
   <div class="section" id="usage-notes">
-<h1>Usage notes<a class="headerlink" href="#usage-notes" title="Permalink to this headline">¶</a></h1>
+<h1>Usage notes</h1>
 <div class="toctree-wrapper compound">
 </div>
 <div class="section" id="action-notes">
-<h2>Action Notes<a class="headerlink" href="#action-notes" title="Permalink to this headline">¶</a></h2>
-<p>Notes on Actions.</p>
+<h2>Action Notes</h2>
+<p>
+  <ul>
+    <li>
+      Below actions will use v2 REST API <a href="https://docs.netskope.com/en/rest-api-v2-overview-312207.html" target="_blank">endpoints
+      </a> if <b>v2 API Key</b> is configured in the asset. Actions will continue to support v1 API Key if v2 API Key is not provided, but it is recommended to use v2 API Key.
```

Comment:
Change v1 and v2 to V1 and V2 as the parameter name displayed on UI has V1 and V2

Reply:
Done